debian/debathena-pyhesiodfs.postinst

```diff
@@ -25,8 +25,17 @@ set -e
 
 case "$1" in
     configure)
+
+    # Adding pyhesiodfs only if the local user group fuse exists.
+    # The group fuse no longer exists in jessie.
+    # If the group fuse is needed, it should have been added when the package 
+    # fuse, a dependency, was configured, before this script is run.
+    if grep "^fuse:" /etc/group > /dev/null 2>&1; then
         adduser pyhesiodfs fuse
-        
+    else
+        echo "The group fuse does not exist on your system. Skipping adding pyhesiodfs to it." >&2
+    fi
```

Comment:
This script apparently has mixed tab and space indenting... I did not discover it at first as I had set tab length to 4. I'm not sure what style Debathena follows, but I'm just going to turn all the tab space into spaces, as it doesn't affect most lines.